.github/workflows/check_pr.yml

```
@@ -0,0 +1,53 @@
+name: Verilator PR ci
+
+on: push
+
+jobs:
+    vlt:
+        runs-on: ubuntu-18.04
+        name: build and test
+        env:
+            TRAVIS_OS_NAME: linux
+            CCACHE_COMPRESS: 1
+            CCACHE_DIR: ${{ github.workspace }}/.ccache
+            CC: ccache gcc
+            CXX: ccache g++
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+            - name: Cache
+              uses: actions/cache@v2
+              env:
+                  cache-name: ccache
+              with:
+                  path: ${{ github.workspace }}/.ccache
+                  key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ github.ref }}-${{ github.sha }}
+                  restore-keys: |
+                      ${{ runner.os }}-build-${{ env.cache-name }}-
+                      ${{ runner.os }}-build-
+                      ${{ runner.os }}-
+            - name: Build
+              env:
+                  TRAVIS_BUILD_STAGE_NAME: build
```

```suggestion
                  CI_BUILD_STAGE_NAME: build
```

Can you please include converting the variable names to CI_ so these will be less confusing and more general.